packages/api-explorer/src/components/SideNav/MethodBadge.tsx

```diff
@@ -6,50 +31,38 @@ interface MethodBadgeProps {
   verb: 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH' | 'TRACE' | 'HEAD'
 }
 
+function getMethodColor(verb: string) {
+  switch (verb) {
+    case 'DELETE':
+      return 'critical'
+    case 'GET':
+      return 'inform'
+    case 'HEAD':
+      return 'neutral'
+    case 'PATCH':
+      return 'warn'
+    case 'POST':
+      return 'positive'
+    case 'PUT':
+      return 'key'
+    case 'TRACE':
+      return 'warn'
+    default:
+      return 'neutral'
+  }
+}
+
 const StyledMethodBadge = styled(Badge)`
+  border: 1px solid transparent;
+  border-radius: 4px;
   display: block;
   font-size: 0.5rem;
-  margin-right: ${(props) => props.theme.space.xsmall};
+  margin-right: ${({ theme }) => theme.space.xsmall};
   padding: 0;
   text-align: center;
   min-width: 2.5rem;
```

Comment:
Making this many changes on top of an existing @looker/component opens up the opportunity for a lot of breakage (if we change implementation details of our `Badge` component. This might be a candidate for a new component from us or a "snowflake" here.